regulations/static/regulations/css/scss/partials/_sidebar_left.scss

```diff
@@ -29,11 +29,49 @@ aside.left-sidebar {
 
 		li {
 			list-style: none;
+			overflow: hidden;
+			max-height: 500em;
 
 			a {
 				text-decoration: none;
 				font-weight: bold;
 			}
+
+			&[data-state="collapsed"] {
+				max-height: 0;
+			}
+
+			transition: all .5s ease-out;
+		}
+
+		button {
+			border: none;
+			background: none;
+			color: $reverse_link_color;
+			cursor: pointer;
+			outline: none;
+
+			&:hover {
+				color: $reverse_link_hover;
+				@include transition;
+			}
+
+			&[data-set-state="expanded"][data-state="expanded"] {
+				display: none;
+			}
+			&[data-set-state="expanded"][data-state="collapsed"] {
+				display: initial;
+			}
+			&[data-set-state="collapsed"][data-state="expanded"] {
+				display: initial;
+			}
```

Comment:
I think these can be removed since this is the default being overridden by the other `display:none` rules.